src/main/java/seedu/address/model/item/VisualizeList.java

```diff
@@ -62,8 +74,9 @@ public Item remove(int index) {
      * Removes an item from the list.
      * @param item the item to be removed from the list
      */
-    public void remove(Item item) {
+    public Item remove(Item item) {
         list.remove(item);
+        return item;
```

Comment:
Do you want to rename your method? I believe this will override the remove method implementation in ModifiableObservableList.

Reply:
have not encountered an issue yet, but renaming is probably a good idea.